src/ogma/features/nodeGrouping.ts

```diff
@@ -218,7 +217,29 @@ export class NodeGroupingTransformation {
     );
   }
 
-  private _unpinNode(node: Node<LkNodeData, LkEdgeData>): void {
-    void node.setAttribute('layoutable', true);
+  private _unpinNode(nodes: Array<NodeList | null>): void {
+    nodes.forEach((nodeList) => {
+      if (nodeList !== null) {
+        void nodeList.setAttribute('layoutable', true);
```

Comment:
I see a small risk here of not awaiting. Since you run the layout just after, what happens if the unpining happens after layout is ran?
maybe it's not an issue, but if it is, could you change this using `map` and `Promise.all`?

Reply:
I don't get the part of using map.
change it to promise all and we need to await before running the layout

Reply:
I added another comment explaining my thought, but it's optional